src/main/java/codechicken/nei/recipe/GuiRecipeTabs.java

```diff
@@ -192,4 +196,22 @@ protected boolean mouseClicked(int x, int y, int mouseButton) {
         }
         return false;
     }
+
+    protected boolean mouseScrolled(int i) {
+        Point mPos = getMousePosition();
```

Comment:
Minor: We are moving away from prefixes. I'd rename it to just `pos`.

Reply:
I see. Then you now know that its very hard for outsiders to read it as mousePosition and writing it out would've been cleaner :kekw: